lib/protobuf/wire/varint.ex

```diff
@@ -138,7 +138,7 @@ defmodule Protobuf.Wire.Varint do
           x6::7, 1::1, x7::7, 1::1, x8::7, 0::1, x9::7>>
       end,
       quote(generated: true) do
-        band(
+        v =
```

Comment:
Would doing `_ = ` here directly still fix the warning?

Reply:
I did try that, but it didn't work!